apiserver/admin_test.go

```diff
@@ -54,9 +54,15 @@ func (s *baseLoginSuite) SetUpTest(c *gc.C) {
 	if s.ControllerConfigAttrs == nil {
 		s.ControllerConfigAttrs = make(map[string]interface{})
 	}
-	s.ControllerConfigAttrs[corecontroller.JujuManagementSpace] = "mgmt01"
+
 	s.JujuConnSuite.SetUpTest(c)
 	loggo.GetLogger("juju.apiserver").SetLogLevel(loggo.TRACE)
+
+	_, err := s.State.AddSpace("mgmt01", "", nil, false)
+	c.Assert(err, jc.ErrorIsNil)
+
+	err = s.State.UpdateControllerConfig(map[string]interface{}{corecontroller.JujuManagementSpace: "mgmt01"}, nil)
```

Comment:
Can't use `ControllerConfigAttrs` for this because anymore because `SetupTest` sets API host ports before the space has been added.